phpstan.neon.dist

```diff
@@ -43,3 +43,6 @@ parameters:
             path: '*/fragments/*'
         - '#Unsafe usage of new static\(\)\.#'
         - '#Constructor of class rex_form_.*_element has an unused parameter \$tag.#'
+        -
+            message: '#.*deprecated.*#'
+            path: '*/update.php'
```

Comment:
in update.php dateien erlauben wir weiterhin deprecated calls, da wir dort zum teil unter alten redaxo core versionen laufen, wo ggf. neue methoden/funktionen noch nicht enthalten sind